src/pages/Create.jsx

```diff
@@ -19,43 +22,102 @@ function Create() {
   const { isMute } = React.useContext(AudioContext)
   const { manifest, characterManager } = React.useContext(SceneContext)
   const [ classes, setClasses ] = useState([]) 
-  
+  let [walletCollections, setWalletCollections] = useState(null)
+
+  useEffect(()=>{
+      if (walletCollections == null){
+        setWalletCollections (new WalletCollections()); 	  
+      }
+    },[])
   useEffect(() => {
+
     if (manifest?.characters != null){
-      const manifestClasses = manifest.characters.map((c) => {
+
+      const manifestClasses = getCharacterManifests(getAsArray(manifest.characters));
+      const nonRepeatingCollections = [];
+      const seenCollections = new Set();
+      manifest.characters.forEach((c) => {
+      if (c.collectionLock != null && !seenCollections.has(c.collectionLock)) {
+          nonRepeatingCollections.push(c.collectionLock);
+          seenCollections.add(c.collectionLock);
+        }
+      });
+      setClasses(manifestClasses);
+
+    }
+  }, [manifest])
+
+  const back = () => {
+    setViewMode(ViewMode.LANDING)
+    !isMute && playSound('backNextButton');
+  }
+
+  const getCharacterManifests = (charactersArray) =>{
+      return charactersArray.map((c) => {
+        let enabled = c.collectionLock == null ? false : true;
         return {
           name:c.name, 
           image:c.portrait, 
           description: c.description,
           manifest: c.manifest,
           icon:c.icon,
           format:c.format,
-          disabled:false
+          disabled:enabled,
```

Comment:
this made me chuckle

Maybe change `let enabled = c.collectionLock == null ? false : true;` to `let disabled= c.collectionLock == null ? true : false;`